CryptoTechReminderSystem.Test/TestDouble/GetLateDevelopersSpy.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using CryptoTechReminderSystem.Boundary;
+using CryptoTechReminderSystem.UseCase;
+
+namespace CryptoTechReminderSystem.Test.TestDouble
+{
+    public class GetLateDevelopersSpy : IGetLateDevelopers
+    {
+        public bool Called { private set; get; }
+
+        public GetLateDevelopersResponse Execute()
+        {
+            Called = true;
+            
+            return new GetLateDevelopersResponse
+            {
+                Developers = new List<string>
+                {
+                    "Uncle Craig"
```

Comment:
who is uncle craig - don't like this one bit

Reply:
We can change this - it's just test data we made up

Reply:
:-1:

Reply:
😖